bindings/python/justfile

```
@@ -1,29 +1,26 @@
 set fallback
-VENV := env_var_or_default("VENV", "../../.venv")
+VENV := env_var_or_default("VENV", ".venv")
 VENV_BIN := VENV / "bin"
 
 @venv:
   python3 -c "import virtualenv" || python3 -m pip --quiet install virtualenv
   python3 -m virtualenv {{VENV}} --quiet
-
-## Set up virtual environment and install requirements
-@requirements: venv
-  {{VENV_BIN}}/python -m pip --quiet install --upgrade pip
-  {{VENV_BIN}}/pip --quiet install -r requirements.txt
+  {{VENV_BIN}}/python -m pip install --upgrade pip
+  {{VENV_BIN}}/pip install -r requirements.txt
 
 ## Compile and install py-glaredb for development
-build *args: requirements
+build *args: venv
   @unset CONDA_PREFIX
-  {{VENV_BIN}}/maturin develop {{args}}
+  {{VENV_BIN}}/maturin develop --pip-path="{{VENV_BIN}}/pip" --extras="pandas,polars,pyarrow" {{args}}
 
-test: requirements
-  {{VENV_BIN}}/pytest -s
+test: venv
+  {{VENV_BIN}}/pytest --rootdir={{invocation_directory()}}/bindings/python
```

this way it'll work regardless of the directory you are in. 

```suggestion
  {{VENV_BIN}}/pytest --rootdir={{justfile_directory()}}
```